src/material-luxon-adapter/BUILD.bazel

```diff
@@ -0,0 +1,56 @@
+package(default_visibility=["//visibility:public"])
+
+load("//tools:defaults.bzl", "ng_module", "ng_package", "ng_web_test_suite", "ng_test_library")
+load("//:packages.bzl", "ROLLUP_GLOBALS")
+
+ng_module(
+  name = "material-luxon-adapter",
+  srcs = glob(["**/*.ts"], exclude=["**/*.spec.ts"]),
+  module_name = "@angular/material-luxon-adapter",
+  deps = [
+    "@angular//packages/core",
+    "@matdeps//@types/luxon",
+    "@matdeps//luxon",
+    "//src/lib/core",
+  ],
+  # Explicitly specify the tsconfig that is also used by Gulp. We need to explicitly use this
+  # tsconfig because in order to import Luxon with TypeScript, we need some special options
+  # enabled.
+  tsconfig = ":tsconfig-build.json",
```

Comment:
Maybe my commenting wasn't too clear, but we shouldn't need this here because Luxon doesn't need the `synthetic` imports mentioned in the `tsconfig-build.json` of the moment adapter package.